transports/transport-highway/src/test/java/org/apache/servicecomb/transport/highway/TestHighwayTransport.java

```diff
@@ -51,6 +52,8 @@ public void testInit() {
       status = false;
     }
     Assert.assertTrue(status);
+
+    VertxUtils.blockCloseVertxByName("transport");
```

Comment:
I think we could change the code to put the blockCloseVertxByName code into finally block to close vertx even there is an exception is thrown.
  ```
 } finally {
      VertxUtils.blockCloseVertxByName("transport");
 }   
```

Reply:
not a problem?
because UT failed, CI will terminate at once